src/sentry/newsletter/base.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+
+
+class Newsletter(object):
+    PRODUCT_CAMPAIGN = 1
```

Comment:
Just making up shit here, but this seemed reasonable to me. Correct me if you want something different @ehfeng.